### PR TITLE
Provide more info in host Orchestrator REST API

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java [hunk 1]
```diff
@@ -2,27 +2,48 @@
...
+    public static final String FIELD_NAME_APPLICATION_URL = "applicationUrl";
+    public static final String FIELD_NAME_SERVICES = "services";
 
     private final String hostname;
     private final String state;
+    private final String applicationUrl;
+    private final List<HostService> services;
+
+    // Deprecated - kept for backwards compatibility until clients have migrated away
+    public GetHostResponse(String hostname, String state) {
```
Comment (bjorncs) [inline orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java]: Consider adding a deprecation annotation

**Review by bjorncs — APPROVED** [review]

File: orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java (hunk 1 above)
Comment (hakonhall) [reply orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java]: Aren't warnings treated as errors? I plan to remove this method later today, once I have updated the other repos.

Comment (bjorncs) [reply orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java]: Yes, but you can annotate all methods using the deprecated feature with SupressWarning("deprecation").

Comment (hakonhall) [reply orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java]: OK, good to know.

**Review by bjorncs — APPROVED** [review]

**Pull request merged**